xml/schema/types/lights-4-19-2.xsd

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="utf-8"?>
+<?xml-stylesheet href="../schema2xhtml.xsl" type="text/xsl"?>
+
+<!-- This schema is part of JMRI. Copyright 2009.                           -->
+<!--                                                                        -->
+<!-- JMRI is free software; you can redistribute it and/or modify it under  -->
+<!-- the terms of version 2 of the GNU General Public License as published  -->
+<!-- by the Free Software Foundation. See the "COPYING" file for a copy     -->
+<!-- of this license.                                                       -->
+<!--                                                                        -->
+<!-- JMRI is distributed in the hope that it will be useful, but WITHOUT    -->
+<!-- ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or  -->
+<!-- FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License  -->
+<!-- for more details.                                                      -->
+
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema"
+           xmlns:xsi ="http://www.w3.org/2001/XMLSchema-instance"
+           xmlns:docbook="http://docbook.org/ns/docbook"
+           xmlns:jmri="http://jmri.org/xml/schema/JMRIschema"
+           xsi:schemaLocation="
+                http://jmri.org/xml/schema/JMRIschema http://jmri.org/xml/schema/JMRIschema.xsd
+                http://docbook.org/ns/docbook http://jmri.org/xml/schema/docbook/docbook.xsd
+            "
+        >
+
+    <xs:complexType name="LightManagerType">
+      <xs:annotation>
+        <xs:documentation>
+          Define the XML stucture for storing the contents of a LightManager implementation.
+        </xs:documentation>
+        <xs:appinfo>
+            <jmri:usingclass configurexml="true">jmri.managers.configurexml.AbstractLightManagerConfigXML</jmri:usingclass>
+        </xs:appinfo>
+      </xs:annotation>
+      <xs:sequence>
+        
+        <xs:element name="light" type="LightType" minOccurs="0" maxOccurs="unbounded" />
+        
+      </xs:sequence>
+      <xs:attribute name="class" type="classType" use="required"/>
+    </xs:complexType>
+
+
+   <xs:complexType name="LightType">
+            <xs:sequence>
+      <xs:element name="systemName" type="lightSystemNameType" minOccurs="1" maxOccurs="1"/>
+      <xs:group ref="CommonNamedBeanContentElements"/>
+              <xs:element name="lightcontrol" minOccurs="0" maxOccurs="unbounded" >
+                <xs:complexType>
+           		  <xs:attribute name="controlType" type="xs:string" use="required"/>
```

Comment:
This line has a TAB character which makes indentation wrong.

Reply:
Fixed, thanks.